### DISCUSSION TITLE
Installing 2.19

### DISCUSSION THREAD

**Discussion by Trippyx87** [EssentialsX/Essentials#4414] [Q&A] [answered]
Just had a question if I upload this to my minecraft server will it read what I have off of the 2.18.2 I currently have or will I have to start over?

**Comment by Chew** [comment, marked as answer]
Yes, we don't intentionally delete any data; you'll be able to upgrade fine. As always, we recommend you take a backup regardless.

**Comment by Trippyx87** [comment]
So to update do I just upload it via ftp? Do I delete the 2.18.2 version I have?

**Reply by Chew** [reply]
Yes, delete all EssentialsX-whatever.jars and replace them
